engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -325,63 +325,26 @@ abstract class CromwellTestkitSpec(val twms: TestWorkflowManagerSystem = new Cro
     }
   }
 
-  /**
-   * Performs the following steps:
-   *
-   * <ol>
-   * <li> Sends the specified message to the implicitly passed `ActorRef` via an `ask`.
-   * <li> Collects the `Future[Any]` response.
-   * <li> Downcasts the `Future[Any]` to a `Future[M]`.
-   * <li> Issues a blocking `Await.result` on the `Future`, yielding an `M`.
-   * </ol>
-   *
-   */
-  // FIXME: This is currently only used in commented out specs
-  def messageAndWait[M: ClassTag](message: AnyRef)(implicit actorRef: ActorRef): M = {
-    val futureAny = actorRef.?(message)(timeout = timeout, sender = actorRef)
-    Await.result(futureAny.mapTo[M], TimeoutDuration)
-  }
-
-  /**
-   * Wait for exactly one occurrence of the specified info pattern in the specified block within a limited amount of
-   * time. The block is in its own parameter list for usage syntax reasons.
-   */
-  // FIXME: Used here & SimpleWorkflowActorSpec
-  def waitForPattern[T](pattern: String, occurrences: Int = 1)(block: => T): T = {
-    within(TimeoutDuration) {
-      waitForInfo(pattern, occurrences) {
-        block
-      }
-    }
-  }
-
   private def buildCromwellRootActor(config: Config) = {
     TestActorRef(new TestCromwellRootActor(config), name = "TestCromwellRootActor")
   }
 
-  def workflowSuccessFilter = EventFilter.info(pattern = "transition from FinalizingWorkflowState to WorkflowSucceededState", occurrences = 1)
-
   def runWdl(sampleWdl: SampleWdl,
-             eventFilter: EventFilter = workflowSuccessFilter,
              runtime: String = "",
              workflowOptions: String = "{}",
              terminalState: WorkflowState = WorkflowSucceeded,
              config: Config = DefaultConfig)(implicit ec: ExecutionContext): Map[FullyQualifiedName, WdlValue] = {
     val rootActor = buildCromwellRootActor(config)
     val sources = WorkflowSourceFiles(sampleWdl.wdlSource(runtime), sampleWdl.wdlJson, workflowOptions)
-    val id = eventFilter.intercept {
-      within(TimeoutDuration) {
-        val workflowId = rootActor.underlyingActor.submitWorkflow(sources)
-        verifyWorkflowState(rootActor.underlyingActor.serviceRegistryActor, workflowId, terminalState)
-        getWorkflowOutputsFromMetadata(workflowId, rootActor.underlyingActor.serviceRegistryActor)
-      }
-    }
-
+    val workflowId = rootActor.underlyingActor.submitWorkflow(sources)
+    eventually { verifyWorkflowState(rootActor.underlyingActor.serviceRegistryActor, workflowId, terminalState) }
+    val outcome = getWorkflowOutputsFromMetadata(workflowId, rootActor.underlyingActor.serviceRegistryActor)
     system.stop(rootActor)
-    id
+    // And return the outcome:
+    outcome
   }
 
-  def eventually[T](isFatal: Throwable => Boolean, maxRetries: Int = 3)(f: => T): T = {
+  def cromwellTestkitSpecEventually[T](isFatal: Throwable => Boolean, maxRetries: Int = 3)(f: => T): T = {
```

Comment:
Hmm, I had to rename this to avoid a naming collision with `Eventually`... 
I guess I could give it a synonym, `tryUntilTestPasses` or something...